pkg/vere/disk.c

```diff
@@ -1020,6 +1027,14 @@ c3_o u3_disk_epoc_init(u3_disk* log_u) {
     }
   }
 
+  // write the metadata to the database
+  if ( c3y == eps_o ) {
+    if ( c3n == u3_disk_save_meta(log_u->mdb_u, who_d, fak_o, lif_w) ) {
+      fprintf(stderr, "disk: failed to save metadata\r\n");
+      exit(1);
+    }
+  }
+
```

Comment:
this is fine for now, but we're going to get the metadata out of arvo going forward. we can save that change for the end, or for a future version of the epoch format.